### DISCUSSION TITLE
Keep constant height

### DISCUSSION THREAD

**Discussion by dzek69** [gpbl/react-day-picker#1440] [Support] [answered]
Currently the height of the rendered element changes depending on month.

This causes:
- content below the picker to jump - sometimes it's not desired, but not a real pain
- the calendar itself jumping up and down when it's displayed in centered modal for example, which is ultra annoying when one wants to press buttons to change months fast several times (the buttons sometimes jumps out of the cursor and you have to move the mouse to continue clicking)

My proposal is to add some way of keeping the height constant, without having to manually wrap it with a div with defined height that I have to calculate myself

**Comment by gpbl** [comment, marked as answer]
> having to manually wrap it with a div with defined height that I have to calculate myself

This is actually the desired solution. Another way is to use fixed weeks as explained here: https://react-day-picker.js.org/basics/customization#using-fixed-weeks

**Reply by dzek69** [reply]
Fixed weeks should do the job, I forgot about it. Days in previous/next months have `rdp-day_outside` class, so I can easily apply `visibility: hidden` to hide them but keep the space.

I don't like the idea of wrapping div, because having to manually sum up heights of elements that may differ from theme to theme is troublesome and such things always leads to bugs later.

Thanks for the answer.
